static/static.js

```diff
@@ -280,7 +280,10 @@ $("#greyOutUnderNav").click(function() {
 
 var lastH3 = null;
 var bounceAnimTO = null;
-$(window).scroll(function() {
+
+// HACK: This debounce is bad but I can't figure out a cleaner solution without a major refactor
+// See issue https://github.com/donejs/bit-docs-donejs-theme/issues/48
+$(window).scroll(new Debounce(function(e) {
```

Comment:
Calling this as a constructor function seems kinda weird because there’s no object that needs to be kept; the variables within the closure can keep the state without instantiating a new object.